lib/portlayer/exec/container.go

```diff
@@ -47,7 +47,7 @@ import (
 type State int
 
 const (
-	StateStarting = iota + 1
+	StateStarting State = iota
```

Comment:
iirc @caglar10ur specifically asked for the iota + 1 (in this or another change) --> I can't recall the reasoning...might double check